src/weather_magic/ui.cljs

```diff
@@ -16,27 +16,38 @@
   [hidden-or-not]
   (hidden-or-not {:hidden :visible :visible :hidden}))
 
+(defn toggle-play-stop
+  ""
+  [atom key]
+  (if (:play-mode (key @atom)) (swap! atom assoc-in [key :play-mode] false) (swap! atom assoc-in [key :play-mode] true)))
```

Comment:
Visst går detta skriva kortare med typ att swappa med (not ((:play-mode (key @atom)) eller liknande?